code/src/utils/calculate-cart-items-counter/CalculateCartItemsCounter.test.ts

```diff
@@ -0,0 +1,39 @@
+import { CartItem } from "@/types/cart.types";
+import calculateCartItemsCounter from "@/utils/calculate-cart-items-counter/CalculateCartItemsCounter";
+
+const generateCart = (length: number, increaseQuantity: boolean = false) => {
+  return Array.from({ length }, (_, i) => ({
+    productId: i + 1,
+    quantity: increaseQuantity ? i + 1 : 1
+  })) as unknown as CartItem[];
+};
```

Comment:
suggest to avoid some `unknown` just pass the type for arguments of fn
like 
```jsx
const generateCart = (length: number, increaseQuantity: boolean = false): CartItem[] => {}
```

Reply:
I guess I need `unknown` there because `{ productId: number, quantity: number }` does not satisfy `CartItem` type. Just writing type like `(length: number, increaseQuantity: boolean = false): CartItem[]` leads to ts error in return statement. I could write entire cart item object, but there is no much sense since in this function only `quantity` property is needed